web/frontend/src/components/base/index.svelte

```diff
@@ -287,6 +287,18 @@
     angle = event.detail.value;
   };
 
+  const handleOnKeyUpRun = (event: KeyboardInput) => {
+    if (event.key === 13) {
```

Comment:
I'd rather check against the `code` of `"Enter"`, it won't require people having to look up what `key` maps to `13`